json-form-react/utilities/JsonFormValidator.js

```diff
@@ -110,4 +115,20 @@ export default class JsonFormValidator {
     return null;
   }
 
+  _regex(condition, value) {
+    const regexCondition = new RegExp(condition.replaceAll('/', ''));
+    if (regexCondition.test(value)) {
+      return null;
+    } else {
+      return 'The value for __FIELD_NAME__ should contain only letters and spaces';
+    }
+  }
+
+  _accepted(value) {
```

Comment:
@raghuhvarman Let's consolidate this into `required` validation.

Reply:
Fixed